schemas/1.0/general/applicant.json

```diff
@@ -8,6 +8,7 @@
   "properties": {
     "date_of_birth": { "type": "string", "format": "date" },
     "nino": { "type": ["string", "null"] },
+    "benefit_type": {"type": "string", "enum": ["universal_credit", "guarantee_pension", "jsa", "esa", "income_support", "none"] },
```

Comment:
We may need `null` here too, if the attribute comes with `nil` value.

Reply:
Yes, of course, my bad. Thank you!